Client/Frontend/Widgets/LoginTableViewCell.swift

```diff
@@ -55,6 +72,19 @@ class LoginTableViewCell: UITableViewCell {
         return imageView
     }()
 
+    /// Override the default accessibility label since it won't include the description by default
+    /// since it's a UITextField acting as a label.
+    override var accessibilityLabel: String? {
+        get {
+            return "\(highlightedLabel.text ?? ""), \(descriptionLabel.text ?? "")"
```

Comment:
Looks like in VoiceOver the secureTextInput fields say:

'Secure text field is editing - 10 characters, insertion point is in...'

In this case, I won't append the description label when secureTextEntry is TRUE.